docs/how-to-add-cypress-tests.md

```diff
@@ -0,0 +1,44 @@
+# How to add Cypress tests
+
+When making changes to JavaScript, CSS, or HTML which could change the functionality or layout of a page, it's important to add corresponding [Cypress](https://docs.cypress.io) integration tests.
+
+To learn how to write Cypress tests, or 'specs', please see Cypress' official [documentation](https://docs.cypress.io/guides/getting-started/writing-your-first-test.html).
+
+> Note: When writing tests for freeCodeCamp, remember to add `/* global cy */` to the top of the file to avoid ESLint issues.
+
+### Where to add a test
+
+- Cypress tests are in the `./cypress` directory.
+
+- Cypress tests for a curriculum module are in the corresponding curriculum directory, i.e. `cypress/integration/learn/responsive-web-design/basic-css/index.js`.
+
+### How to run tests
+
+**1. Ensure that MongoDB and client applications are running**
+
+  - [Start MongoDB and seed the database](/how-to-setup-freecodecamp-locally?id=step-3-start-mongodb-and-seed-the-database)
```

Comment:
```suggestion
  - [Start MongoDB and seed the database](/how-to-setup-freecodecamp-locally#step-3-start-mongodb-and-seed-the-database)
```
Sorry, I missed this the first time.  If you use anchor links, Docsify automatically converts them to id query params as needed.  At the moment it's fine either way, but `?id=` is an implementation detail and could break in future.